solidity/contracts/Chainlinked.sol

```diff
@@ -143,18 +142,18 @@ contract Chainlinked {
 
   function completeChainlinkFulfillment(bytes32 _requestId)
     internal
-    checkChainlinkFulfillment(_requestId)
+    chainlinkFulfillment(_requestId)
   {}
 
-  modifier checkChainlinkFulfillment(bytes32 _requestId) {
-    require(msg.sender == unfulfilledRequests[_requestId], "source must be the oracle of the request");
-    delete unfulfilledRequests[_requestId];
+  modifier chainlinkFulfillment(bytes32 _requestId) {
```

Comment:
I think this modifier would be more clear if it included a verb to indicate that an action is taken.

Proposal: `recordChainlinkFulfillment` and `fulfillChainlinkRequest` for the modifier and method. They aren't very different, but the modifier and the method are also not very different. Thoughts?